ante/fee.go

```diff
@@ -90,6 +109,21 @@ func IsOracleOrGravityTx(msgs []sdk.Msg) bool {
 	return true
 }
 
+// AssertMinProtocolGasPrice returns an error if the provided gasPrices are lower then
+// the required by protocol.
+func AssertMinProtocolGasPrice(gasPrices sdk.DecCoins) error {
```

Comment:
Assert sounds like a method for unit tests, what about `CheckMinGasPrice`?

Reply:
For me check sounds like something to return boolean. There are two schools:
* Must* panics -- this is what is in std lib and my personal preference
* Assert* panics -- this is what is usually in other languages.

What do you think?

Reply:
yes, though this function doesn't panic, hence I'm using `Assert`.